package.json

```diff
@@ -17,9 +17,13 @@
   },
   "dependencies": {
     "code-push": "1.5.2-beta",
-    "semver": "^5.1.0"
+    "semver": "^5.1.0",
+    "rnpm": "1.4.1"
   },
-  "devDependencies": {
-    "react-native": "0.19.0"
+  "peerDependencies": {
+    "react-native": "^0.19.0"
+  },
+  "scripts": { 
+    "postinstall": "cd ../../ && rnpm link react-native-code-push"
```

Comment:
It is definitely supposed to no-op and show you this message - 

```
rnpm info Linking react-native-code-push android dependency 
rnpm info Android module react-native-code-push is already linked 
rnpm info Linking react-native-code-push ios dependency 
rnpm info iOS module react-native-code-push is already linked 
```

That said, there's some weird issue with android where it is inserting the same lines in gradle twice.